gen/gtk_consts_gen.jl

```diff
@@ -1,13 +1,22 @@
+const PLATFORM_SPECIFIC = Dict{String, Any}(
+    "G_DIR_SEPARATOR"           => :(Sys.iswindows() ? '\\' : '/'),
+    "G_DIR_SEPARATOR_S"         => :(Sys.iswindows() ? "\\" : "/"),
```

Comment:
The path separator is `Base.Filesystem.path_separator`.

Reply:
Thanks for the info!